app/views/session_answers/coronavirus_find_support/_need_help_with.html.erb

```diff
@@ -2,15 +2,15 @@
   What do you need help with because of coronavirus?
 <% end %>
 
-<%= form_with path: session_flow_path(flow_name, node_name), method: :put, local: true do |form| %>
+<%= form_for_node do %>
   <%= render "govuk_publishing_components/components/checkboxes", {
```

Comment:
Looking good 👍  Just wondering why you've not created a `govuk_checkbox_for_node` method?

Reply:
Just the way I work ... I look to put something into a shared pattern the second or third time they are used. As this is only used once so far, I haven't created a shared method. I'd look to move it into a shared help once there are more than one forms that need check boxes.

Reply:
No, problem.  I thought, as we know we'll need 2 checkbox questions - :need_help_with and :nation you might have side-stepped the `remove duplication` step and gone straight to the end game.